internal/control_plane/db/etcd/client.go

```diff
@@ -49,19 +51,19 @@ func (client *etcdClient) DeleteKey(ctx context.Context, id string) (bool, error
 //PutValue puts the given key-value pair in etcd database
 func (client *etcdClient) PutValue(ctx context.Context, key string, value string) error {
 	_, err := client.db.Put(ctx, key, value)
-	return err
+	return octerr.New(3, err)
 
 }
 
 //GetValue gets the value of the given key
 func (client *etcdClient) GetValue(ctx context.Context, id string) (string, error) {
 	res, err := client.db.Get(ctx, id)
 	if err != nil {
-		return "", err
+		return "", octerr.New(3, err)
 	}
 	gr := res.OpResponse().Get()
 	if len(gr.Kvs) == 0 {
-		return "", errors.New("no value found")
+		return "", octerr.New(3, errors.New("no value found"))
```

Comment:
I guess it will be better if you add `no value found` as a constant in errors file as it is being used 3 times in the same file

Reply:
NoValueFound constant is added.